jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/FormFieldsSection.java

```diff
@@ -0,0 +1,31 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.complex.*;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.*;
+
+public class FormFieldsSection {
+    @UI("#simple-form-field-input")
```

Comment:
Why aren't you using smart locators. It's simple and also reduces the number of lines of code.

Reply:
Changed the variables to use smart locators.